Mathlib/LinearAlgebra/Matrix/ToLin.lean

```diff
@@ -1008,3 +1008,39 @@ def algEquivMatrix [Fintype n] (h : Basis n R M) : Module.End R M ≃ₐ[R] Matr
 #align alg_equiv_matrix algEquivMatrix
 
 end
+
+namespace Basis
+
+variable {R M ι : Type*} [CommRing R] [AddCommGroup M] [Module R M] [Fintype ι] [DecidableEq ι]
```

Comment:
Am I right in thinking that we don't have the basis for linear maps? If so, how about:
```suggestion
variable {R M ι : Type*} [CommRing R] [AddCommGroup M] [Module R M] [Fintype ι] [DecidableEq ι]
  {ι' M' : Type*} [AddCommGroup M'] [Module R M'] [Fintype ι'] [DecidableEq ι']

def linearMap (b : Basis ι R M) (b' : Basis ι' R M') : Basis (ι' × ι) R (M →ₗ[R] M') :=
  (Matrix.stdBasis R ι' ι).map (LinearMap.toMatrix b b').symm
```
and `end` below can be an `abbrev` or even maybe absent?

Reply:
Fixed. I opted to have the abbrev, and copy the tiny API. I think it aids discoverability and readability in applications.